jerry-ext/include/jerryscript-ext/arg.h

```diff
@@ -64,6 +64,12 @@ jerry_value_t jerryx_arg_transform_args (const jerry_value_t *js_arg_p,
                                          const jerryx_arg_t *c_arg_p,
                                          jerry_length_t c_arg_cnt);
 
+jerry_value_t jerryx_arg_transform_object_properties (const jerry_value_t taregt,
+                                                      const jerry_char_t **name_p,
+                                                      const jerry_length_t name_cnt,
+                                                      const jerryx_arg_t *c_arg_p,
+                                                      jerry_length_t c_arg_cnt);
```

Comment:
Do we need two counters? (name_cnt and c_arg_cnt)? Why one of them is const and the other is not? Btw const is not needed for non pointer types.